content/influxdb/cloud-iox/visualize-data/grafana.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Use Grafana to visualize data
+seotitle: Use Grafana to visualize data stored in InfluxDB
+list_title: Grafana
+description: >
+  Use [Grafana](https://grafana.com/) to query and visualize data stored in an
+  InfluxDB bucket backed by InfluxDB IOx.
+weight: 101
+menu:
+  influxdb_cloud_iox:
+    name: Grafana
+    parent: Visualize data
+influxdb/cloud-iox/tags: [visualization]
+---
+
+Use [Grafana](https://grafana.com/) to query and visualize data stored in an
+InfluxDB bucket backed by InfluxDB IOx.
+
+> [Grafana] enables you to query, visualize, alert on, and explore your metrics,
+> logs, and traces wherever they are stored.
+> [Grafana] provides you with tools to turn your time-series database (TSDB)
+> data into insightful graphs and visualizations.
+>
+> {{% caption %}}[Grafana documentation](https://grafana.com/docs/grafana/latest/introduction/){{% /caption %}}
+
+For the most performant queries, use SQL and the
+[Flight SQL protocol](https://arrow.apache.org/blog/2022/02/16/introducing-arrow-flight-sql/)
+to query InfluxDB.
+
+
+## Build the Grafana FlightSQL plugin
+
+{{% warn %}}
+The Grafana FlightSQL plugin is experimental and is subject to change.
+{{% /warn %}}
+
+### Install requirements
+
+- [Yarn](https://classic.yarnpkg.com/lang/en/docs/install/)
+- [Mage](https://magefile.org/)
+- [Grafana](https://grafana.com/grafana/download) 9.2.0+
+
+### Build the FlightSQL plugin from source
+
+1.  Clone the Grafana FlightSQL plugin repository:
+
+    ```bash
+    git clone https://github.com/influxdata/grafana-flightsql-datasource.git
+    ```
+
+2.  Navigate into the clone repo:
+
+    ```sh
+    cd grafana-flightsql-datasource
+    ```
+
+3.  Use `yarn` to build the FlightSQL plugin from source:
+
+    ```sh
+    yarn build-plugin
+    ```
+
+    <div id="flightsql-plugin-path"></div>
+
+4.  Once built, navigate into the newly created `dist` directory in your
+    `grafana-flightsql-datasource` repository.
+    Use `pwd` to return the currently directory path. You will need this later.
+
+    ```
+    cd dist
+    pwd
+    ```
```

Comment:
We can switch this out now as our [release](https://github.com/influxdata/grafana-flightsql-datasource/releases) pipeline is working which should simplify the steps here, @brettbuddin is going to supply the suggested changes here.